Sources/SwiftDEBot/Utils/User+Handle.swift

```diff
@@ -1,8 +1,7 @@
-import Sword
+import DiscordBM
 
-extension User {
-    /// Use this to mention the user, evaluates to `"<@!\(self.id)>"`. No need for additional @s.
+extension PartialUser {
     var mentionHandle: String {
-        return "<@!\(self.id)>"
```

Comment:
This seems to be fine, but i think Discord has deprecated @-ing people with this syntax. better to use `"<@\(self.id)>"` (no `!`) or use `DiscordUtils.userMention(id: self.id)`